compiler/z/codegen/OMRLinkage.hpp

```diff
@@ -567,9 +566,8 @@ enum TR_DispatchType
    virtual TR::RealRegister::RegNum getPrivateStaticBaseRegister()        { return _privateStaticBaseRegister; }
    virtual TR::RealRegister *getPrivateStaticBaseRealRegister();
 
-   virtual TR::RealRegister::RegNum setExtCodeBaseRegister (TR::RealRegister::RegNum r)   { return _extCodeBaseRegister = r; }
-   virtual TR::RealRegister::RegNum getExtCodeBaseRegister()    { return _extCodeBaseRegister; }
-   virtual TR::RealRegister *getExtCodeBaseRealRegister();
+   // TODO (#2848): Once the downstream dependency is removed we can fold this setter
+   virtual TR::RealRegister::RegNum setExtCodeBaseRegister (TR::RealRegister::RegNum r)   { return TR::RealRegister::GPR7; }
```

Comment:
We'll clean this up right after this PR is merged. I'll have the downstream dependency PR up soon.

Reply:
https://github.com/eclipse/openj9/pull/3419